nengo_gui/static/menu.js

```diff
@@ -27,6 +27,8 @@ Nengo.Menu.prototype.show = function (x, y, items) {
         return;
     }
 
+    this.menu_margin = 15;
+
```

Comment:
(basically, I'd argue that you want it to go right to the edge -- there's no reason to have a buffer there, so I'm curious as to what platforms put a buffer there and if we can figure out why)